src/cloud-priv.yml

```diff
@@ -76,6 +76,32 @@ components:
       $ref: './common/schemas/Links.yml'
     OrganizationRequest:
       $ref: './cloud/schemas/OrganizationRequest.yml'
+    BucketLimits:
+      $ref: "./quartz/schemas/BucketLimits.yml"
+    CheckLimits:
+      $ref: "./quartz/schemas/CheckLimits.yml"
+    DashboardLimits:
+      $ref: "./quartz/schemas/DashboardLimits.yml"
+    NotificationEndpointLimits:
+      $ref: "./quartz/schemas/NotificationEndpointLimits.yml"
+    NotificationRuleLimits:
+      $ref: "./quartz/schemas/NotificationRuleLimits.yml"
+    OrganizationWithToken:
+      $ref: "./quartz/schemas/OrganizationWithToken.yml"
+    Organization:
+      $ref: "./quartz/schemas/Organization.yml"
+    Organizations:
+      $ref: "./quartz/schemas/Organizations.yml"
+    OrgLimits:
+      $ref: "./quartz/schemas/OrgLimits.yml"
+    RateLimits:
+      $ref: "./quartz/schemas/RateLimits.yml"
+    RestrictedLimit:
+      $ref: "./quartz/schemas/RestrictedLimit.yml"
+    TaskLimits:
+      $ref: "./quartz/schemas/TaskLimits.yml"
+    Unlimited:
+      $ref: "./quartz/schemas/Unlimited.yml"
```

Comment:
You need to also add something for `./quartz/schemas/Limit.yml` to remove the `2` that is throwing off oats

Reply:
It should be able to be named differently if `Limit` is already taken

Reply:
made the change since Stacie's out for the weekend (and off next week)